buildSrc/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-instrumentation.gradle.kts

```diff
@@ -1,10 +1,13 @@
 plugins {
   id("io.opentelemetry.instrumentation.javaagent-testing")
   id("io.opentelemetry.instrumentation.muzzle-check")
+  id("io.opentelemetry.instrumentation.javaagent-codegen")
```

Comment:
Seems to come in transitively (though doesn't hurt at all either)

Reply:
Transitivity via `javaagent-testing` is very obscure. I prefer to have obvious dependencies.